release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -24,8 +24,9 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
-//TODO: Add this for 4.14
-Starting with {product-title} 4.12, an additional six months is added to the Extended Update Support (EUS) phase on even numbered releases from 18 months to two years. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Starting with {product-title} 4.12, Red Hat has added a 6-month Extended Update Support (EUS) phase for even numbered releases, extending the total available lifecycle from 18 months to 2 years. In the {product-title} 4.12 release, this additional EUS is only available for the `x86_64` architecture.
+
+Starting with {product-title} 4.14, Red Hat has added an optional 12-month additional EUS Term 2, extending the total available lifecycle from 2 years to 3 years. The additional EUS Term 2 is available on all architecture variants of {product-title}. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
 
 Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to 64-bit ARM, {ibm-power-name} (ppc64le), and {ibm-z-name} (s390x) platforms. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

Comment:
```suggestion
Starting with {product-title} 4.12, Red Hat has added a 6-month Extended Update Support (EUS) phase for even numbered releases, extending the total available lifecycle from 18 months to 2 years. In the {product-title} 4.12 release, this additional EUS is only available for the `x86_64` architecture. Starting with {product-title} 4.14, this additional EUS is extended to 64-bit ARM, {ibm-power-name} (ppc64le), and {ibm-z-name} (s390x) platforms. 

Starting with {product-title} 4.14, Red Hat has added an optional 12-month additional EUS Term 2, extending the total available lifecycle from 2 years to 3 years. The additional EUS Term 2 is available on all architecture variants of {product-title}. 

For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```
@jaypoulz I think if we restructure like this it should be correct also for Arm/P/Z

Reply:
I believe this is correct but it should really be ACKed by Duncan.

Reply:
Thanks, Silke. Updated as per your suggestion.

Reply:
LGTM

Reply:
Apologies, should be using this account for approvals